package.json

```diff
@@ -24,14 +24,12 @@
     "fluent-langneg": "0.1.0",
     "fluent-react": "0.4.1",
     "html-react-parser": "0.3.5",
-    "isomorphic-fetch": "2.2.1",
     "js-cookie": "2.1.4",
-    "l20n": "5.0.0",
```

Comment:
Looks like this is still used [in Storybook config](https://github.com/mozilla/testpilot/blob/d43e00498a5ffcc3b8cb9a272e56f035e954db98/.storybook/config.js#L2), though it shouldn't actually do anything since the switch over to fluent. 

Also see a `require('l20n')` over in [`frontend/src/scripts/locale.js`](https://github.com/mozilla/testpilot/blob/d43e00498a5ffcc3b8cb9a272e56f035e954db98/frontend/src/scripts/locale.js#L53).

Reply:
Thanks!

This branch was supposed to remove frontend/src/scripts/locale.js, whoops...

Reply:
Well, it turns out we can't quite remove locale.js yet, but we should be able to when we switch to putting the language at the beginning of the url.